rbac/management/group/view.py

```diff
@@ -47,6 +47,9 @@
 VALID_EXCLUDE_VALUES = ['true', 'false']
 VALID_GROUP_ROLE_FILTERS = ['role_name', 'role_description']
 VALID_GROUP_PRINCIPAL_FILTERS = ['principal_username']
+ORDERING_PARAM = 'order_by'
+VALID_ROLE_ORDER_FIELDS = ['name', 'modified', 'policyCount']
```

Comment:
Also, curious if since we have to name the filters prepended with `role_` and `principal_` if we shouldn't do the same here on the ordering for consistency on nested resources? Open to suggestions there.

Reply:
Hmm, that's a good thought. The ordering param is generic, though, where the sorting fields will depend on what's being sorted. Maybe a way to import the sorting fields would be best, so they're not sitting in the group codebase, but that doesn't really alter how the ordering_param is set, but may kick the decision can down the road a bit?